src/provider.ts

```diff
@@ -78,7 +78,7 @@ export class DelphinusBrowserConnector extends DelphinusProvider<BrowserProvider
       throw "MetaMask not installed, Browser mode is not available.";
     }
     // https://eips.ethereum.org/EIPS/eip-1193#summary
-    super(new BrowserProvider(window.ethereum));
+    super(new BrowserProvider(window.ethereum, "any"));
```

Comment:
https://docs.ethers.org/v6/api/providers/#BrowserProvider
From the doc, this parameter is used to force the network? so originally if we do not put anything, it is generating exception when we switch network? It is weird for me as we had add switch network functionality for our browserConnector and I guess it should work originally?